Demo/Measures-cms/AdvancedIllnessandFrailty-1.8.000.cs

```diff
@@ -94,6 +94,51 @@ public AdvancedIllnessandFrailty_1_8_000(CqlContext context)
 
     #endregion
 
+	private CqlCode[] LOINC_Value()
+	{
+		var a_ = new CqlCode[0]
```

Comment:
This looks wrong, in the old version there were codes returned here, but now it is empty.